kuma/javascript/src/task-completion-survey.jsx

```diff
@@ -120,7 +120,7 @@ export default function TaskCompletionSurvey(props: Props) {
                 nonInteraction: true
             });
         }
-    }, [shouldShow]);
+    }, [shouldShow, ga]);
```

Comment:
It is harmless to add ga here, but not actually necessary since it is known to be constant, so here the linter is giving us a less useful signal.